examples/src/Example09BasicMutation.elm

```diff
@@ -0,0 +1,77 @@
+module Example09BasicMutation exposing (main)
+
+import CustomScalarCodecs
+import Graphql.Document as Document
+import Graphql.Http
+import Graphql.Operation exposing (RootMutation)
+import Graphql.SelectionSet as SelectionSet exposing (SelectionSet)
+import Helpers.Main
+import RemoteData exposing (RemoteData)
+import Swapi.Enum.Phrase exposing (Phrase(..))
+import Swapi.Mutation as Mutation exposing (SendMessageRequiredArguments)
+import Swapi.Object as Object exposing (ChatMessage)
+import Swapi.Object.ChatMessage as ObjChat
+
+
+type alias ChatMessage =
+    { phrase : Phrase
+    }
+
+
+type alias Response =
+    Maybe ChatMessage
+
+
+chatSelection : SelectionSet ChatMessage Object.ChatMessage
+chatSelection =
+    SelectionSet.map ChatMessage
```

Comment:
The record seems unnecessary with just a single value. How about also adding the character, like in this example:

https://github.com/dillonkearns/elm-graphql/blob/master/examples/subscription/Main.elm#L34-L37